contracts/governance/VoteToken.sol

```diff
@@ -104,7 +104,7 @@ abstract contract VoteToken is ERC20, IVoteToken {
         uint256 _value
     ) internal virtual override {
         super._transfer(_from, _to, _value);
-        _moveDelegates(delegates[_from], delegates[_to], uint96(_value));
+        _moveDelegates(delegates[_from], delegates[_to], safe96(_value, "StkTruToken: uint96 overflow"));
```

Comment:
can we write a test for this?

Reply:
I can do that. Note that none of the _other_ safe96 casts have corresponding tests - shall I add tests for all of them or just this new one?

Reply:
I think it's fine to leave it like this.